blueocean-github-pipeline/nginx.conf

```diff
@@ -0,0 +1,15 @@
+daemon off;
+events {
+  worker_connections  1024;
+}
+http {
+  server {
+    listen 9000;
+    location /api/v3/ {
+      proxy_pass https://api.github.com/;
+      sub_filter_once off;
+      sub_filter_types application/json
+      sub_filter https://api.github.com/ http://localhost:9000/api/v3/;
```

Comment:
@michaelneale I had a crack at using the `sub_filter` to substitute data in the response body. Couldn't seem to get this to work. Have you had any experience with `proxy_pass` and `sub_filter`?